physionet-django/events/forms.py

```diff
@@ -60,3 +60,35 @@ class EventApplicationResponseForm(forms.ModelForm):
     class Meta:
         model = EventApplication
         fields = ('status', 'comment_to_applicant')
+
+
+class EventAgreementForm(forms.ModelForm):
+    class Meta:
+        model = EventAgreement
+        fields = (
+            'name',
+            'version',
+            'slug',
+            'is_active',
+            'html_content',
+            'access_template',
+        )
+        labels = {'html_content': 'Content'}
+        help_texts = {
+            'name': '* The displayed name of the agreement.',
+            'version': '* The version number of the agreement.',
+            'slug': '* A simple string for use in the URL displaying the agreement. '
+                    'Should include the version number.',
+            'is_active': '* Only active agreements are usable in future events.',
+            'html_content': '* The agreement text displayed to the participant.',
+            'access_template': '* Instructions on accessing the dataset.'
+        }
+
+    def clean(self):
+        cleaned_data = super(EventAgreementForm, self).clean()
+        if EventAgreement.objects.filter(name=cleaned_data['name'], version=cleaned_data['version']).exists():
+            raise forms.ValidationError(
+                {"name": ["Agreement with this name and version already exists for this user."],
```

Comment:
This is great, although because the model is no longer unique for the user, we should update the text to remove the mention of for this user:

```suggestion
                {"name": ["An agreement with this name and version already exists."],
```